core-java-sample/src/main/java/org/spine3/sample/Requests.java

```diff
@@ -65,44 +63,37 @@ public static CommandRequest addOrderLine(UserId userId, OrderId orderId) {
                 .setTitle("Domain Driven Design.")
                 .setPrice(price)
                 .build();
-
         final int quantity = 1;
         final double totalPrice = book.getPrice() * quantity;
-
         final OrderLine orderLine = OrderLine.newBuilder()
                 .setProductId(Messages.toAny(book.getBookId()))
                 .setQuantity(quantity)
                 .setTotal(totalPrice)
                 .build();
-
-        final AddOrderLine cmd = AddOrderLine.newBuilder()
+        final AddOrderLine command = AddOrderLine.newBuilder()
                 .setOrderId(orderId)
                 .setOrderLine(orderLine).build();
-
-        final CommandRequest result = CommandRequest.newBuilder()
-                .setCommand(Messages.toAny(cmd))
-                .setContext(getCommandContext(userId)).build();
-        return result;
+        return newCommandRequest(userId, command);
     }
 
     public static CommandRequest payForOrder(UserId userId, OrderId orderId) {
-
         final BillingInfo billingInfo = BillingInfo.newBuilder().setInfo("Payment info is here.").build();
-
-        final PayForOrder cmd = PayForOrder.newBuilder()
+        final PayForOrder command = PayForOrder.newBuilder()
                 .setOrderId(orderId)
                 .setBillingInfo(billingInfo)
                 .build();
+        return newCommandRequest(userId, command);
+    }
 
-        final CommandRequest result = CommandRequest.newBuilder()
-                .setCommand(Messages.toAny(cmd))
-                .setContext(getCommandContext(userId))
-                .build();
-        return result;
+    public static CommandRequest newCommandRequest(UserId userId, Message command) {
+        final CommandContext context = getCommandContext(userId);
+        final CommandRequest.Builder request = CommandRequest.newBuilder()
```

Comment:
As requested in another pull request, please have a utility method for building a CommandRequest by command and context. There should be such a method already.